sale_commercial_partner/tests/test_sale_commercial_partner.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2018 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import Form, common
+
+
+class TestSaleCommercialPartner(common.TransactionCase):
+    def setUp(self):
```

Comment:
Thanks for adding the test! Can you refactor to use @classmethod setUpClass(cls)?